### DISCUSSION TITLE
The structure of your repo

### DISCUSSION THREAD

**Discussion by jinhan26** [clarenceluo78/deep-learning-lookup#5] [Q&A] [answered]
Hi! May I ask what's the structure, i.e. your idea of organizing your lookup project? I notice that there are some folder with `code` prefix and some folder with `trick` prefix, I wonder if there is any specific definition?

**Comment by clarenceluo78** [comment, marked as answer]
Thanks for your answer! As you noticed, there are in total three types of folders in my repo. The first is the utility function folder (as well as a python package) `util`. The second is the code folders with prefix `code`, which contain the code for different implementations (from scratch or using API) of deep learning and machine learning algorithms in python scripts or jupyter notebooks. The third is the code folders with prefix `trick`, which mainly contain some tricks used in machine learning deployment such as dropout, model selection trick, etc. 

Hope this answer helps!
